packages/unmock-core/src/service/spy/types.ts

```diff
@@ -0,0 +1,24 @@
+import { SinonSpy as SinonSpyType } from "sinon";
+import { ISerializedRequest, ISerializedResponse } from "../../interfaces";
+import { ISpyDecoration } from "./generated";
+
+export type SinonSpy = SinonSpyType<[ISerializedRequest], ISerializedResponse>;
+
+export interface ServiceSpy extends SinonSpy, ISpyDecoration {} // tslint:disable-line:interface-name
```

Comment:
Using an interface instead of `type ServiceSpy = ...` seems more user-friendly as interfaces can be extended.

Reply:
Well I guess nowadays one can extend type aliases as well, using an interface is only better here if someone wants to do declaration merging 🤔